### PR TITLE
[Update] Update the about page

### LINKED ISSUE #30: [BA2250 > Volunteering in BA2250] Clarify what "outside of these [standard 8am-9pm] hours" means on the Volunteering page for future reference.
"Members of the Executive Council shall have access outside of these hours to the lounge (BA2250) and storage room (BA2285)."

Someone asks, "Do execs get 24/7 lounge access (excluding UofT holidays)?"
An exec wrote, "not sure, i think so"

Could a current exec or past exec please clarify (including a screenshot of exec's quote, or paraphrased response) whether "outside of standard hours" imply 24/7 lounge access, excluding UofT holidays and other closures?

Cheers :D

### LINKED ISSUE #74: [Update] Update executive council details on "About" page
### Description

Currently waiting for headshots from 4 out of 6 people.
Right now I have the changes already made, just need to commit and push. Waiting for #71 to be finalized.

### PR DESCRIPTION
**Pull request by JasonBarahan**
Update the about page to reflect newly appointed executives. 

Resolves #30 
Resolves #74

### CODE REVIEW

**Comment by JasonBarahan** [comment]
Latest commit closes #30

File: content/about/index.mdx [hunk 1]
```diff
@@ -58,4 +51,7 @@ The CSSU proudly maintains a student lounge in BA 2250 in which students can han
 
 At the CSSU, we strive to represent Computer Science students as best as possible, and to create an inclusive and welcoming community. As part of this effort, we are committed to listening to the feedback of CSSU members. Using this always-open form, you can submit feedback to us about any aspect of our activities, past or present. If you would like, you can provide your name or contact email so we can follow up with you directly. You may also remain anonymous. All feedback is welcome! [**Share your thoughts!**](https://forms.gle/bb4JXfWYkqW7Ewhh8)
 
+Alternatively, you can take a look at suggestions that have been made by other people [**here:**](https://github.com/cssu/suggestions/issues)
```
Comment (bbayazit16) [inline content/about/index.mdx]: . instead of : ?

**Review by bbayazit16 — COMMENTED** [review]
Looks good, thank you!

**Review by bbayazit16 — DISMISSED** [review]

**Review by dannychen27 — APPROVED** [review]

**Comment by dannychen27** [comment]
do you want to merge all of these commits to main, baris?

**Pull request merged**